subprojects/testfx-core/src/main/java/org/testfx/matcher/control/ComboBoxMatchers.java

```diff
@@ -0,0 +1,137 @@
+import javafx.scene.control.Labeled;
+import javafx.scene.control.TableView;
+
+import org.hamcrest.Factory;
+import org.hamcrest.Matcher;
+import org.testfx.api.annotation.Unstable;
+
+import static org.testfx.matcher.base.GeneralMatchers.typeSafeMatcher;
+
+@Unstable(reason = "needs more tests")
+public class ComboBoxMatchers {
+
+    //---------------------------------------------------------------------------------------------
+    // STATIC METHODS.
+    //---------------------------------------------------------------------------------------------
+
+    @Factory
+    @Unstable(reason = "is missing apidocs")
+    public static Matcher<Node> hasItems(int amount) {
+        String descriptionText = "has " + amount + " items";
+        return typeSafeMatcher(ComboBox.class, descriptionText, node -> hasItems(node, amount));
+    }
+
+    @Factory
+    @Unstable(reason = "is missing apidocs")
+    public static <T> Matcher<Node> hasSelection(T selection) {
+        String descriptionText = "has selection " + selection;
+        return typeSafeMatcher(ComboBox.class, descriptionText, node -> hasSelection(node, selection));
+    }
+
+    @Factory
+    @Unstable(reason = "is missing apidocs")
+    public static <T> Matcher<Node> containsItems(T... items) {
+        String descriptionText = "contains items " + Arrays.toString(items);
+        return typeSafeMatcher(ComboBox.class, descriptionText, node -> containsItems(node, items));
+    }
+
+    @Factory
+    @Unstable(reason = "is missing apidocs")
+    public static <T> Matcher<Node> containsExactlyItems(T... items) {
+        String descriptionText = "contains exactly items " + Arrays.toString(items);
+        return typeSafeMatcher(ComboBox.class, descriptionText, node -> containsExactlyItems(node, items));
+    }
+
+    @Factory
+    @Unstable(reason = "is missing apidocs")
+    public static <T> Matcher<Node> containsItemsInOrder(T... items) {
+        String descriptionText = "contains items in order " + Arrays.toString(items);
+        return typeSafeMatcher(ComboBox.class, descriptionText, node -> containsItemsInOrder(node, items));
+    }
+
+    @Factory
+    @Unstable(reason = "is missing apidocs")
+    public static <T> Matcher<Node> containsExactlyItemsInOrder(T... items) {
+        String descriptionText = "contains exactly items in order " + Arrays.toString(items);
+        return typeSafeMatcher(ComboBox.class, descriptionText, node -> containsExactlyItemsInOrder(node, items));
+    }
+
+    //---------------------------------------------------------------------------------------------
+    // PRIVATE STATIC METHODS.
+    //---------------------------------------------------------------------------------------------
+
+    private static boolean hasItems(ComboBox<?> comboBox,
+                                    int amount) {
+        return comboBox.getItems().size() == amount;
+    }
+
+    private static <T> boolean hasSelection(ComboBox<?> comboBox,
```

Comment:
Alternatively this could be `hasSelectedItem()`. But `hasSelection()` is equally adequate.